SG14/inplace_function.h

```diff
@@ -214,6 +217,26 @@ class inplace_function<R(Args...), Capacity, Alignment>
         ::new (std::addressof(storage_)) C{std::forward<T>(closure)};
     }
 
+    template<size_t Cap, size_t Align>
+    inplace_function(const inplace_function<R(Args...), Cap, Align>& rhs)
+        : inplace_function(rhs.vtable_ptr_, rhs.vtable_ptr_->copy_ptr, std::addressof(rhs.storage_))
```

Comment:
I invariably prefer to see parens rather than curlies, except when giving an initializer-list (`vector<int> v = {1,2,3};`) or for some reason when naming a tag type (`std::monostate{}`). For certain types, parens do something different from curlies. However, since in this case parens and curlies do the same thing, I guess I don't mind using curlies.

Reply:
Actually, I take it back: here we're not just whatever-initializing a member, we're creating a delegating constructor. I'd really prefer to keep `inplace_function(...)` here.